Makefile

```diff
@@ -156,9 +156,9 @@ e2e-coverage:
 kind-load: $(KIND) #EXHELP Loads the currently constructed image onto the cluster.
 ifeq ($(CONTAINER_RUNTIME),podman)
```

Comment:
Probably out of scope for what this PR is trying to accomplish (and I asked in the other now-closed PR), but would be possible to use this podman procedure for `docker` as well? That way we don't have to maintain two different implementations of kind-load.

Reply:
I would be find doing that, I can just remove the ifeq bits and leave only the first block with the temp archive. That should always work regardless of if the user is using podman, docker, or podman with the "podman-docker" alias package.